packages/psss/src/containers/UpcomingReportCard.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import { Warning } from '@material-ui/icons';
+import { CardContent, CardFooter, CardHeader, BarMeter, Card, Button } from '@tupaia/ui-components';
+import { openWeeklyReportsPanel, setActiveWeek } from '../store';
+import {
+  getWeekNumberByPeriod,
+  getFormattedStartByPeriod,
+  getFormattedEndByPeriod,
+} from '../utils';
+import { useUpcomingReport } from '../api/queries';
+import { REPORT_STATUSES, DUE_ISO_DAY } from '../constants';
+
+const StyledButton = styled(Button)`
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+`;
+
+const DateSubtitle = styled(Typography)`
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  font-weight: 400;
+  color: ${props => props.theme.palette.text.secondary};
+`;
+
+const getDisplayDays = days => {
+  const d = Math.abs(days);
+  return `${Math.abs(d)} day${d > 1 ? 's' : ''}`;
```

Comment:
No need to use `Math.abs` twice:

```suggestion
  const d = Math.abs(days);
  return `${d} day${d > 1 ? 's' : ''}`;
```